bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -166,22 +177,37 @@ for trigger_file, injection_file in zip(args.trigger_files,
                  (len(found_within_time), len(missed_within_time)))
 
     logging.info('Removing injections in vetoed time')
+
+    # Put individual detector vetoes into list of all vetoed indices, if it
+    # is vetoed in more than one detector, then it will be repeated
     vetoid = numpy.array([])
     for ifo in ifo_list:
         vi, _ = indices_within_segments(inj_time, [args.veto_file], ifo=ifo,
                                         segment_name=args.segment_name)
         vetoid = numpy.append(vetoid, vi)
         del vi
+
+    # Find out which of these indices are unique, and how many times each
+    # vetoed index occurs - this is how many detectors it is vetoed in
+    vetoid_unique, count_vetoid = numpy.unique(vetoid, return_counts=True)
+
+    # Vetoed for all combinations are ones which do not allow *any*
+    # combination of min_required_ifos detectors, and so veto more than
+    # (len(ifo_list) - args.min_required_ifos) detectors
```

Comment:
not sure why you call it 'vetoed for all combinations'. this line is deciding which injections to remove, yes? 
so i would say `remove injections where the number of unvetoed ifos is less than the minimum specified by user`

& imo arranging the arithmetic as (len(ifo_list) - count_vetoid) < args.min_required_ifos may be clearer

Reply:
I think that my thought process was more along the lines of 'calculate the number of ifos which would need to be vetoed in order to remove, and then check' - the end result is the same but I've changed it to be clearer